roles/redfish_monitor/tasks/main.yml

```diff
@@ -0,0 +1,40 @@
+---
+- name: Pull Docker image.
+  docker_image:
+    source: pull
+    name: "{{ redfish_monitor_docker_image }}"
+    force_source: yes
+
+- name: Create config directory.
+  file:
+    path: "{{ redfish_monitor_config_path }}"
+    state: directory
+
+- name: Create log directory.
+  file:
+    path: "{{ redfish_monitor_log_folder }}"
+    state: directory
+
+- name: Configure redfish-monitor env file
+  template:
+    src: redfish-monitor.env.j2
+    mode: 0600
+    dest: "{{ redfish_monitor_env_file }}"
+
+- name: Start redfish-monitor docker container
+  docker_container:
+    name: redfish_monitor
+    auto_remove: yes
+    image: "{{ redfish_monitor_docker_image }}"
+    network_mode: host
+    env_file: "{{ redfish_monitor_env_file }}"
+    volumes:
+      - "{{ redfish_monitor_log_folder }}:/var/log"
```

Comment:
`/var/log/` is kind of generic: is it possible that other logs would accidentally be written there, that we don't want? Like, syslog normally goes there, or /var/log/messages, which we presumably don't want in particular. Though, it might not be an issue for application containers, which don't run with an init system.